### DISCUSSION TITLE
请问官方有计划出一个@formily/ant-design-vue这样的组件库吗

### DISCUSSION THREAD

**Discussion by chuan602** [alibaba/formily#2014] [Q&A 2.x] [answered]
如题，请问官方是否有计划支持ant-design-vue，出一个基于ant-design-vue和formily的组件库，提供@formily/antd类似的组件，如布局组件、自增组件等。
目前表单设计器生成的代码或json配置文件都是适用于@formily/antd，对于使用ant-design-vue组件库的用户不够便捷。

**Comment by janryWang** [comment, marked as answer]
vue生态目前都是社区共建的

**Reply by faeli** [reply]
同样遇到此情况，可以参考官方提供的 element 实现 ant-design-vue 组件库
